### PR TITLE
Travis support for better code review

### PR DESCRIPTION
**Pull request by Mic92**
This is complementary for github review
as it provides instead feedback in the pr interface.
Since all it does is building release.nix the
maintenance effort should minimal.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+language: nix
+script: nix build -f release.nix build.x86_64-linux
```
Comment (Mic92) [inline .travis.yml]: I had to use this instead of `nix-build` because it produced too much output.

**Comment by Mic92** [comment]
There is currently I am investigating since it works on my nixpkgs revision: https://travis-ci.com/Mic92/nixops/jobs/156237797#L4428

```console
$ python2 -c 'import boto.ec2'
```

File: .travis.yml (hunk 1 above)
Comment (Mic92) [reply .travis.yml]: Found the `nix-build --quiet` option.

**Comment by Mic92** [comment]
Builds now: https://travis-ci.com/Mic92/nixops/builds/90943057

**Comment by Mic92** [comment]
Builds now https://travis-ci.com/Mic92/nixops/builds/90943180

File: nixops/diff.py [hunk 2]
```diff
@@ -163,7 +183,9 @@ def _default_handle(self):
         raise NotImplementedError
 
     def get_deps(self):
+        # type: () -> List[Handler]
         return self._dependencies
 
     def get_keys(self,*keys):
+        # type: (*Any) -> List[str]
```
Comment (Mic92) [inline nixops/diff.py]: Strings can be str and unicode here?

File: nixops/diff.py [hunk 3]
```diff
@@ -143,7 +163,7 @@ def retrieve_def(d):
 
 class Handler(object):
     def __init__(self, keys, after=None, handle=None):
-        # type: (Dict[str, Dict[str,str]], Optional[List], Optional[Callable]) -> None
+        # type: (List[str], Optional[List], Optional[Callable]) -> None
```
Comment (Mic92) [inline nixops/diff.py]: same question here

**Comment by domenkozar** [comment]
lgtm

**Comment by domenkozar** [comment]
I've enabled travis-ci for this repository.

**Comment by domenkozar** [comment]
Mosts tests are functional, so you need configured backends.

**Comment by tomberek** [comment]
Add `typing` to release.nix?

**Comment by Mic92** [comment]
@tomberek see https://github.com/NixOS/nixops/blob/d5ad09cd3db5ed096d1bd17fbe5a0790b434f2c7/release.nix#L99

**Pull request merged**